test/files/ALS/als000/pass-als000-alignment_geometry.ifc

```diff
@@ -0,0 +1,194 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION (('ViewDefinition [Alignment-basedView]'), '2;1');
+FILE_NAME('pass-als000-alignment_geometry.ifc','2023-10-12T15:19:11',('redacted'),('redacted'),'IfcOpenShell 0.8.0','redacted - redacted - 3.14159','');
+FILE_SCHEMA(('IFC4X3_ADD2'));
```

Comment:
```suggestion
FILE_DESCRIPTION(('ViewDefinition[Alignment-basedView]'),'2;1');
FILE_NAME('pass-als000-alignment_geometry.ifc', '2022-09-16T10:35:07', ('Evandro Alfieri'), ('buildingSMART Int.'), 'IfcOpenShell 0.0.0', 'IfcOpenShell contributors - Bonsai - 0.8.2-alpha250225', 'Nobody');
FILE_SCHEMA(('IFC4X3_ADD2'));
```
This should make the header valid 
![image](https://github.com/user-attachments/assets/6b588c4a-cc83-488b-82cd-15ce7288f994)